src/walletSdk/horizon/AccountService.ts

```diff
@@ -0,0 +1,18 @@
+import { Keypair } from "stellar-sdk";
+
+import { SigningKeypair } from "./Account";
+
+// Do not create this object directly, use the Wallet class.
+export class AccountService {
+  private server;
+  private network;
```

Comment:
could we add types for `server` and `network` ?

Reply:
https://github.com/stellar/typescript-wallet-sdk/pull/21/commits/a281475ea5eb914e55a7d616a969882d87aeda3e